mmhuman3d/core/cameras/camera_parameters.py

```diff
@@ -149,6 +149,42 @@ def set_K_R_T(self,
         self.set_mat_np('rotation_mat', R_mat)
         self.set_value('translation', T_vec.tolist())
 
+    def get_K_R_T(self, k_dim=3) -> List[np.ndarray]:
```

Comment:
Shall we use `get_KRT` to replace `get_K_R_T`? Cause (1) `KRT` is easier to be typed out and (2) `KRT` is a widely acceptable convention in camera. @LazyBusyYang @JasonBoy1

Reply:
I think `get_KRT` is much better

Reply:
All get and set methods have been transferred to `_KRT`
https://github.com/open-mmlab/mmhuman3d/pull/60/files#diff-61ad06924613416ad0a2bb2c18575d1754cb449ddc80b932cc9ce154a509b5b1R152